cybershell/plugins.py

```
@@ -0,0 +1,52 @@
+class PluginBase:
+    name = 'PluginBase'
+    def __init__(self, config: SafetyConfig):
+        self.config = config
+    def in_scope(self, target: str) -> bool:
+        return self.config.in_scope(target)
+    def run(self, **kwargs) -> PluginResult:
+        raise NotImplementedError
+
+class HttpFingerprintPlugin(PluginBase):
+    name = 'HttpFingerprintPlugin'
+    def run(self, **kwargs) -> PluginResult:
```

Ruff (0.12.2)

Missing type annotation for `**kwargs`

(ANN003)